unixfs/io/dagreader.go

```diff
@@ -14,17 +14,23 @@ import (
 	ipld "gx/ipfs/Qme5bWv7wtjUNGsK2BNGVUFPKiuxWrsqrtvYwCLRw8YFES/go-ipld-format"
 )
 
-var ErrIsDir = errors.New("this dag node is a directory")
-
-var ErrCantReadSymlinks = errors.New("cannot currently read symlinks")
+// Common errors
+var (
+	ErrIsDir            = errors.New("this dag node is a directory")
+	ErrCantReadSymlinks = errors.New("cannot currently read symlinks")
+)
 
+// A DagReader represents a ReadSeekCloser which offers additional methods
```

Comment:
> represents a ReadSeekCloser which offers additional methods

This doesnt really get at what the dagreader *does*. I would say something more like:

```
// A DagReader provides read-only read and seek acess to a unixfs file.
```

Reply:
Thanks